.circleci/config.yml

```diff
@@ -109,24 +109,24 @@ workflows:
     jobs:
       - run-specs:
           context: slack-secrets
-          name: run-specs-with-postgres
+          name: run-specs-with-postgres-ruby-3-2
```

Comment:
Thoughts on removing defaults and always listing the database and ruby_version?

Reply:
Good idea. Going to update this.